.eslintrc.js

```diff
@@ -11,13 +10,19 @@ module.exports = {
     browser: true,
     node: true,
   },
-  extends: ['plugin:import/recommended', 'airbnb', 'prettier', 'prettier/react'],
-  parser: 'babel-eslint',
+  extends: [
+    'plugin:import/recommended',
+    'plugin:import/typescript',
+    'airbnb-typescript',
+    'prettier',
+    'prettier/react',
+    'prettier/@typescript-eslint',
+  ],
+  parser: '@typescript-eslint/parser',
   parserOptions: {
-    ecmaVersion: 7,
-    sourceType: 'module',
+    // project: './tsconfig.json',
```

Comment:
Is this supposed to be uncommented? If not was is the purpose of this option and why is it not set?

Reply:
This is related to the eslint rules that makes the task to timeout. Let's remove the line, it will be handled by the TODO on the rules that are creating issues.